features/step_definitions/shout_steps.js

```diff
@@ -1,31 +1,33 @@
-const assert = require('assert')
-const { Before, Given, When, Then } = require('cucumber')
-const Shouty = require('../../lib/shouty')
-const Coordinate = require('../../lib/coordinate')
+'use strict'
 
-const ARBITARY_MESSAGE = 'Hello, world'
+const assert = require('assert');
+const { Before, Given, When, Then } = require('cucumber');
+const Shouty = require('../../lib/shouty');
+const Coordinate = require('../../lib/coordinate');
+
+const ARBITARY_MESSAGE = 'Hello, world';
 let shouty
 
 Before(function() {
-  shouty = new Shouty()
-})
+  shouty = new Shouty();
```

Comment:
We don't use semicolons

Reply:
"We don't uses semicolons."

Is that Cucumber Limited, or Javascript developers in general. My recollection of Crockford's advice was that semicolons (and strict) were basic hygiene.

Reply:
Cucumber ltd. 

There are plenty of us devs who do add semi colons. 

JS has automatic semicolon insertion, which works well most of the time. There are some cases where it won’t work.

https://github.com/elwayman02/eslint-es6/blob/master/docs/rules/semi.md

Reply:
Semicolons in JavaScript has largely fallen out of fashion. The cases where they are required are rare.

I think we should avoid them.